component/general/materialComponents/Card2.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import Collapse from '@material-ui/core/Collapse';
+import CardHeader from '@material-ui/core/CardHeader';
+import { withStyles } from '@material-ui/core/styles';
+import CardContent from '@material-ui/core/CardContent';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
+import CardActionArea from '@material-ui/core/CardActionArea';
+
+
+
+const styles = {
+	card: {
+		width: "100%",
+  },
+  expandIcon: {
+    float: "right"
+  }
+};
+
+class NewCard extends React.Component {
+	constructor (props) {
+		super(props);
+		this.state = {
+			expanded: false
+		}
+	}
+
+	render () {
+		const { expanded } = this.state;
+		const { title, subtitle, children, classes } = this.props;
+
+		return (
+			<Card className={classes.card}>
+				<CardActionArea onClick={() => this.setState(oldState => ({ expanded: !oldState.expanded }))}>
+          <CardHeader
+            title={title}
+            subheader={subtitle}
+            avatar={expanded ? <ExpandLessIcon style={{float: "right"}}/> : <ExpandMoreIcon style={{float: "right"}}/>}
```

Comment:
There are three things I'd avoid here: inline style, out of a constant which was already defined above which seems fit, float right, which is usually replaceble by framework classes - layouts. @tarelli it would be nice to have also some guidelines about styling.

Reply:
I am also not a big fan of inlined styles cause the code gets messy quickly, my preference would be style inside the components but not inlined -> https://medium.com/@stephanieworkman/styling-in-react-best-practices-777f6fc5ed69

Reply:
I didn't knew the styled components. Coming from Angular, which has built-in scope-based css, didn't really feel the need for something like that, but in the React context seems really cool. In my view a choice like that should come with consistency. Who starts writing a component - or refactors existing ones- should know that this is the way that style is written, provided that not writing css at all by delegating to material-ui component and specific classes would be the best choice, because opens up the possibility of theming from a central point.

Reply:
card 2 has been deleted